src/handlers/commandHandler.js

```diff
@@ -215,13 +225,15 @@ export default class CommandHandler {
 
   getHelp() {
     const reply = dedent`*Here's what I can do*
-    - \`/stats <Battle Tag> <Region>\`: Retrieve player's stats
-    - \`/links <Battle Tag> <Region>\`: Provides links to websites providing more stats
+    - \`/stats <Battle Tag> <Region / Platform>\`: Retrieve player's stats
+    - \`/links <Battle Tag> <Region / Platform>\`: Provides links to websites providing more stats
 
     *Examples*
     US player: /stats LastBastion#12345
     EU player: /stats LastBastion#12345 eu
     KR player: /stats LastBastion#12345 kr
+    PS4 player: /stats LastBastion#12345 psn
+    Xbox One player: /stats LastBastion#12345 xbl
```

Comment:
Could you remove the battle tag id? I just checked and it seems that console players' battle tags do not have the hash.
```
PS4 player: /stats LastBastion psn
Xbox One player: /stats LastBastion bbl
```